lib/tasks/journeys_report.rake

```diff
@@ -1,65 +1,17 @@
 require 'csv'
 require 'json'
-namespace :journeys do
-  desc 'Exports a CSV report of all journeys'
-  task csv_report: :environment do
-    puts 'exporting journeys_report.csv...'
-
-    CSV.open('journeys_report.csv', 'w') do |csv|
-      csv << ['SUPPLIER', 'DATE', 'MOVE REFERENCE', 'MOVE FROM', 'MOVE TO', 'MOVE EVENTS', 'TIMESTAMP', 'JOURNEY FROM (title)', 'JOURNEY FROM (nomis id)', 'JOURNEY TO (title)', 'JOURNEY TO (nomis id)', 'JOURNEY EVENTS', 'VEHICLE', 'BILLABLE']
-
-      current_reference = nil
-      Journey
-          .joins(:move).includes(:move)
-          .joins(:supplier).includes(:supplier)
-          .joins(:from_location).includes(:from_location)
-          .joins(:to_location).includes(:to_location)
-          .order('suppliers.key ASC, moves.date ASC, moves.reference ASC, journeys.client_timestamp ASC')
-          .each do |journey| # NB: cannot do find_each because we need to order
-        csv << if current_reference != journey.move.reference
-                 # new move
-                 [journey.supplier.key,
-                  journey.move.date,
-                  journey.move.reference,
-                  journey.move.from_location.title,
-                  journey.move.to_location&.title,
-                  journey.move.move_events.map { |event| "#{event.event_name}: #{event.notes}" }.join('; '),
-                  journey.client_timestamp.strftime('%d/%m %H:%M:%S'),
-                  journey.from_location.title,
-                  journey.from_location.nomis_agency_id,
-                  journey.to_location.title,
-                  journey.to_location.nomis_agency_id,
-                  journey.events.map { |event| "#{event.event_name}: #{event.notes}" }.join('; '),
-                  journey.vehicle['registration'],
-                  journey.billable]
-               else
-                 # journey within current move
-                 [nil,
-                  nil,
-                  nil,
-                  nil,
-                  nil,
-                  nil,
-                  journey.client_timestamp.strftime('%d/%m %H:%M:%S'),
-                  journey.from_location.title,
-                  journey.from_location.nomis_agency_id,
-                  journey.to_location.title,
-                  journey.to_location.nomis_agency_id,
-                  journey.events.map { |event| "#{event.event_name}: #{event.notes}" }.join('; '),
-                  journey.vehicle['registration'],
-                  journey.billable]
-               end
-        current_reference = journey.move.reference
-      end
-    end
-  end
+require 'aws-sdk-s3'
 
+namespace :journeys do
   desc 'Exports a JSON report of all journeys'
   task json_report: :environment do
-    puts 'exporting journeys_report.json...'
-
+    timestamp = DateTime.now.utc.strftime('%s')
```

Comment:
```suggestion
    timestamp = Time.zone.now.utc.to_i
```

Reply:
👍

Reply:
@willfish @mafonso `Time.zone.now.utc.to_i` will return the minutes since the unix epoc, not the seconds. I think seconds is probably a safer choice.

Reply:
Oh no, my bad, it _is_ seconds - I got my indentation wrong :-(